GraphSDKGenerator.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>GraphSDKGenerator</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Tool for generating Graph SDKs</description>
+    <releaseNotes></releaseNotes>
+    <copyright>Copyright Microsoft</copyright>
+    <licenseUrl>http://msdn.microsoft.com/en-US/cc300389</licenseUrl>
+    <frameworkAssemblies>
+      <frameworkAssembly assemblyName="System" targetFramework="net45" />
+    </frameworkAssemblies>
+  </metadata>
+  <files>
+    <file src="src\GraphSDKGenerator\bin\Release\GraphSDKGenerator.exe" target="tools" />
+    <file src="src\GraphSDKGenerator\bin\Release\*.dll" target="tools" />
```

Comment:
`src\GraphSDKGenerator\bin\Release\*.dll` is going to include all of those dlls that get generated on the fly for each template. I think we'll want to be explicitly inclusive in this list.

Reply:
It will only include the generated DLLs if we have executed the release version before doing the pack.  Once we move it into a CI build, then we shouldn't have an issue.  Also, we would have to explicitly call out all the 3rd DLLs which is a bit of a pain.  I suggest we put this off for the moment and re-visit it at some point in the near future.